keps/sig-node/3960-pod-lifecycle-sleep-action/kep.yaml

```diff
@@ -0,0 +1,41 @@
+title: "Pod lifecycle sleep action"
+kep-number: 3960
+authors:
+  - "@AxeZhan"
+owning-sig: sig-node
+participating-sigs: []
+status: provisional
+creation-date: 2023-04-22
+reviewers:
+  - "@thockin"
+  - "@bobbypage"
+approvers:
+  - "@sig-node-leads"
```

Comment:
I think you need a named person here.  @SergeyKanzhelev since you have been reviewing?  @mrunalp FYI

Reply:
I can take this one.